src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -502,11 +509,10 @@ private void ValidateWarningsOnHelloWorldApp (PublishCommand publishCommand, Com
             //since the error experience is not good for a developer
```

Comment:
Might be good to change this comment since it no longer talks about the behavior of the code